pyaedt/modeler/cad/Modeler.py

```diff
@@ -1134,6 +1207,388 @@ def _attributes(self):
+            for face in obj.faces:
+                if face.id == self.entity_id:
+                    return obj.name
+                for edge in face.edges:
+                    if edge.id == self.entity_id:
+                        return obj.name
+                    for vertex in edge.vertices:
+                        if vertex.id == self.entity_id:
+                            return obj.name
+        return None  # part has not been found
+
+    @property
+    def _object_parameters(self):
+        """Internally named array with parameters of the object coordinate system."""
+        arg = ["Name:ObjectCSParameters"]
+        _dict2arg(self.props, arg)
+        return arg
+
+    @property
+    def _attributes(self):
+        """Internal named array for attributes of the object coordinate system."""
+        coordinateSystemAttributes = ["NAME:Attributes", "Name:=", self.name, "PartName:=", self._part_name]
+        return coordinateSystemAttributes
+
+    @pyaedt_function_handler()
+    def create(
+        self,
+        cs_type,
+        entity,
+        x_axis_entity=None,
+        move_to_end=True,
+        reverse_x_axis=False,
+        reverse_y_axis=False,
+        # y_axis_x_direction=0,
+        # y_axis_y_direction=0,
+        # y_axis_z_direction=0
+    ):
+        """Create an object coordinate system.
+
+        Parameters
+        ----------
+        cs_type : str
+            Type of Object Coordinate System.
+            Possible types are ``Offset``, ``Rotated``, ``Both``.
+        entity : int, VertexPrimitive, EdgePrimitive, FacePrimitive
+            Entity where the Object Coordinate System is anchored.
+            It can be the Id of the object or an object of type VertexPrimitive, EdgePrimitive, FacePrimitive.
+        x_axis_entity : int, VertexPrimitive, EdgePrimitive, FacePrimitive
+            Entity where the x axis of the Object Coordinate system points to.
+            It can be the Id of the object or an object of type VertexPrimitive, EdgePrimitive, FacePrimitive.
+            x_axis_entity has to be provided only if cs_type is set to ``Both``.
+        move_to_end : bool, optional
+            If ``True`` the Coordinate System creation operation will always be moved to the end of subsequent
+            objects operation. This will guarantee that the coordinate system will remain solidal with the object
+            face. If ``False`` the option "Always Move CS to End" is set to off. The default is ``True``.
+        reverse_x_axis : bool, optional
+            Whether the x-axis is in the reverse direction.
+            The default is ``False``.
+        reverse_y_axis : bool, optional
+            Whether the y-axis is in the reverse direction.
+            The default is ``False``.
+        # y_axis_x_direction : float, optional
+        #     X direction of the y axis.
+        #     This option can be provided if the coordinate system type is either ``Rotated`` or ``Both``.
```

Comment:
```suggestion
        #     This option can be provided if the coordinate system type is either ``"Rotated"`` or ``"Both"``.
```
Assuming the type is a string, the value need to have double back ticks and quotation marks.